README.md

```diff
@@ -4,29 +4,51 @@ A GPU implementation of 2, 3 dimension type 1, 2 non-uniform FFT based on [FINUF
 This is a work from Melody Shih's internship at Flatiron Institute, advised by CCM project leader Alex Barnett.
 
 
-### Installation
- - Get this code and dependency -
+## Installation
+ - Get this code -
  ```git clone https://github.com/flatironinstitute/cufinufft.git```
  - Review the `makefile`. - If you need to customize build settings, create and edit a `make.inc`.  Example:
    - To override the standard CUDA `/usr/local/cuda` location your `make.inc` should contain: ```CUDA_ROOT=/your/path/to/cuda```.
    - Two examples are provided, one for IBM machines (make.inc.power9), and another for the Courant Institute cluster (make.inc.CIMS).
  - Compile - ```make all -j```
  - Run a test code - ``` ./cufinufft2d1_test 2 128 128 10 1e-6```
+ - Many checks can be run using `make check`.
+
+### Library Installation
```

Comment:
I need to update this documentation wrt the expectation of a pypi package for python purposes. That should avoid library conversation except for HPC users.